spotbugs/etc/messages.xml

```diff
@@ -9033,6 +9041,129 @@ Using floating-point variables should not be used as loop counters, as they are
         ]]>
     </Details>
   </BugPattern>
+  <BugPattern type="PI_DO_NOT_REUSE_PUBLIC_IDENTIFIERS_CLASS_NAMES">
+        <ShortDescription>Do not reuse public identifiers from JSL as class name</ShortDescription>
+        <LongDescription>Class name {1} in source file {2} shadows the publicly available identifier from the Java Standard Library.</LongDescription>
+        <Details>
+            <![CDATA[<p>
+      It's essential to avoid reusing public identifiers from the Java Standard Library as class names.
+      This is because the Java Standard Library is a part of the Java platform and is expected to be available in all Java environments.
+      Doing so can lead to naming conflicts and confusion, making it harder to understand and maintain the code.
+      It's best practice to choose unique and descriptive class names that accurately represent the purpose and functionality of your own code.
+      To provide an example, let's say you want to create a class for handling dates in your application. Instead of using a common name like "Date",
+      which conflicts with the existing java.util.Date class, you could choose a more specific and unique name like or "AppDate" or "DisplayDate".
+
+      A few key points to keep in mind when choosing names as identifier:
+      <ul>
+        <li>Use meaningful prefixes or namespaces: Prepend a project-specific prefix or namespace to your class names to make them distinct. For example, if your project is named "MyApp", you could use "MyAppDate" as your class name.</li>
```

Comment:
I was under the impression that [Smurf Naming](https://devcards.io/smurf-naming-convention) is generally discouraged.

Reply:
It also does not belong to my checker. :)